File: app/lib/model/Favourite.dart
class Favourite {
  final String userID;
  final String productID;
  final String productName;
  final String productImage;
  final String productPrice;
  final String productDiscountPrice;
  final bool liked;

  Favourite(this.userID, this.productID, this.productName, this.productImage,
      this.productPrice, this.productDiscountPrice, this.liked);
}

File: API/models/Favourite.js
const mongoose = require("mongoose");
const FavouriteSchema = new mongoose.Schema({
  userID: {
    type: String,
    required: true
  },
  productID: {
    type: String,
    required: true
  },
  productName: {
    type: String,
    required: true
  },
  productImage: {
    type: String,
    required: true
  },
  productPrice: {
    type: String,
    required: true
  },
  productDiscountPrice: {
    type: String,
    required: true
  },
  liked: {
    type: Boolean,
    required: true
  }
});
module.exports = mongoose.model("Favourite", FavouriteSchema);
